src/views/AmalgamationSelection.vue

```diff
@@ -171,20 +175,20 @@ export default class AmalgamationSelection extends Vue {
 
   /** Start Regular Long-form button pressed. */
   async startRegularAmalgamation (): Promise<any> {
-    // let legalType = null
-    // if (this.isBComp) legalType = CorpTypeCd.BC_COMPANY
-    // else legalType = this.getLegalType
-
-    window.alert('This action is not available at the moment. Please check again later.')
+    let legalType = null
+    if (this.isBComp) legalType = CorpTypeCd.BC_COMPANY
+    else legalType = this.getLegalType
 
     try {
       // show spinner since this is a network call
-      this.$root.$emit('showSpinner', true)
-      // const accountId = +JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id || 0
-      // const businessId = await this.createBusinessAA(accountId, legalType)
+      this.setStartingAmalgamationSpinner(true)
+      const accountId = +JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id || 0
+      const businessId = await this.createBusinessAA(accountId, legalType)
```

Comment:
You should pass in the amalgamation type in preparation for horiz/vert.

See also line 215.

Reply:
Looks good. Resolving.